gef-ui/server/types/facility.ts

```diff
@@ -18,7 +18,9 @@ export type Facility = {
   monthsOfCover?: number | null;
   details?: AnyObject | null;
   detailsOther?: string | null;
-  currency?: AnyObject | null;
+  currency?: {
+    id: Currency;
+  } | null;
```

Comment:
This reads funny to me, could/should it be `currency?: Currency;`?

Reply:
Does the currency object have anything else other than id?

This is bizarre, `currency.id = { currency, amount }`

Are we sure it's correct?

Reply:
Just checked staging and it's the same there too

Reply:
Ahhhh I've just noticed - you've looked at the `CurrencyAndAmount` type - this is just `Currency`

Reply:
Ah got it, thanks @MarRobSoftwire !